.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19.12
+          go-version: 1.21.4
```

Comment:
because go-ethereum has been upgraded to 1.21

Reply:
no need, let's keep 1.19

Reply:
```
Error: ../../../go/pkg/mod/github.com/cockroachdb/pebble@v0.0.0-20230928194634-aa077af62593/vfs/mem_fs.go:469:29: y.lockedFiles.Swap undefined (type sync.Map has no field or method Swap)
221note: module requires Go 1.20
```

Reply:
also required in Dockerfile

Reply:
done